### PR TITLE
#106 | Tingting | Read a promotion BDD test

### PR DESCRIPTION
**Pull request by Tingtingyy**

### CODE REVIEW

File: features/promotions.feature [hunk 1]
```diff
@@ -75,3 +75,20 @@ Scenario: Query promotions by Product ID
...
   And I should not see "Grand Theft Auto Trilogy Coupon" in the results
   And I should not see "Chips Ahoy" in the results
+
+Scenario: Read a promotion
+  When I visit the "Home Page"
+  And I set the "Promotion ID" empty
+  And I copy the #1 Promotion ID to the clipboard
+  Then the "Promotion ID" field should be empty
+  When I paste the "Promotion ID" field
+  And I press the "Retrieve" but
```
Comment (dizys) [inline features/promotions.feature:85]: `button` is not spelled completely here.

**Review by dizys — CHANGES_REQUESTED** [review]

**Review by dizys — APPROVED** [review]
LGTM.

**Comment by Tingtingyy** [comment]
close #106

**Comment by dizys** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/555adb64-8497-4119-b8ce-3e3c764c7597?env_id=ibm:yp:us-south "nyu-devops-promotions-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/70ccc3da-a493-45e2-9db5-075c3d858058 "nyu-promotion-service-fall2103") deployed [promotions](https://github.com/PromoSquad/promotions/releases/tag/deploy-ibm-yp-us-south-yk2494-nyu.edu-prod-20211205-214310 "deploy-ibm-yp-us-south-yk2494-nyu.edu-prod-20211205-214310") to [prod](https://console.bluemix.net/apps/05bb5f7f-3579-466f-b3c0-177b78bf73e4?env_id=ibm:yp:us-south "ibm:yp:us-south:yk2494@nyu.edu:prod"), including commit 58999f70a096e37b24e95f2fc07a43f0096de8d5

**Pull request merged**